From 12edb9430a3d902a0e8edb8c3d1489efda8205ad Mon Sep 17 00:00:00 2001
From: Xiong Aifei <xiongaifei@wxiat.com>
Date: Wed, 15 Jun 2022 16:36:00 +0800
Subject: [PATCH] sw64: gpu: replace '_memset_c_io' by 'memset_io'
Patch-mainline: Not yet, from openEuler
References: bsn#22
openEuler-commit: 12edb9430a3d902a0e8edb8c3d1489efda8205ad
Modified-by-SEL: No


Sunway inclusion
category: bugfix
bugzilla: https://gitee.com/openeuler/kernel/issues/I5GDKC

--------------------------------

Signed-off-by: Xiong Aifei <xiongaifei@wxiat.com>

Signed-off-by: Gu Zitao <guzitao@wxiat.com>
Signed-off-by: Guoqing Jiang <guoqing.jiang@suse.com>
---
 drivers/gpu/drm/amd/amdgpu/gfx_v8_0.c | 4 ++--
 drivers/gpu/drm/radeon/radeon_vce.c   | 2 +-
 2 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/drivers/gpu/drm/amd/amdgpu/gfx_v8_0.c b/drivers/gpu/drm/amd/amdgpu/gfx_v8_0.c
index 851d64e83166..28c4e1fe5cd4 100644
--- a/drivers/gpu/drm/amd/amdgpu/gfx_v8_0.c
+++ b/drivers/gpu/drm/amd/amdgpu/gfx_v8_0.c
@@ -1355,7 +1355,7 @@ static int gfx_v8_0_mec_init(struct amdgpu_device *adev)
 		}
 
 #if IS_ENABLED(CONFIG_SW64)
-		_memset_c_io(hpd, 0, mec_hpd_size);
+		memset_io(hpd, 0, mec_hpd_size);
 #else
 		memset(hpd, 0, mec_hpd_size);
 #endif
@@ -4654,7 +4654,7 @@ static int gfx_v8_0_kiq_init_queue(struct amdgpu_ring *ring)
 		mutex_unlock(&adev->srbm_mutex);
 	} else {
 #if IS_ENABLED(CONFIG_SW64)
-		_memset_c_io((void *)mqd, 0, sizeof(struct vi_mqd_allocation));
+		memset_io((void *)mqd, 0, sizeof(struct vi_mqd_allocation));
 #else
 		memset((void *)mqd, 0, sizeof(struct vi_mqd_allocation));
 #endif
diff --git a/drivers/gpu/drm/radeon/radeon_vce.c b/drivers/gpu/drm/radeon/radeon_vce.c
index 68cc5a347d3b..b9680d38d924 100644
--- a/drivers/gpu/drm/radeon/radeon_vce.c
+++ b/drivers/gpu/drm/radeon/radeon_vce.c
@@ -240,7 +240,7 @@ int radeon_vce_resume(struct radeon_device *rdev)
 	}
 
 #ifdef __sw_64__
-	_memset_c_io(cpu_addr, 0, radeon_bo_size(rdev->vce.vcpu_bo));
+	memset_io(cpu_addr, 0, radeon_bo_size(rdev->vce.vcpu_bo));
 #else
 	memset(cpu_addr, 0, radeon_bo_size(rdev->vce.vcpu_bo));
 #endif
-- 
2.33.0

